internal/pkg/cli/deploy/svc.go

```diff
@@ -203,13 +203,14 @@ func (d *workloadDeployer) cachedEnvironmentConfig() (*manifest.Environment, err
 
 // WorkloadDeployerInput is the input to for workloadDeployer constructor.
 type WorkloadDeployerInput struct {
-	SessionProvider *sessions.Provider
-	Name            string
-	App             *config.Application
-	Env             *config.Environment
-	ImageTag        string
-	Mft             interface{} // Interpolated, applied, and unmarshaled manifest.
-	RawMft          []byte      // Content of the manifest file without any transformations.
+	SessionProvider   *sessions.Provider
+	Name              string
+	App               *config.Application
+	Env               *config.Environment
+	ImageTag          string
+	Mft               interface{} // Interpolated, applied, and unmarshaled manifest.
+	RawMft            []byte      // Content of the manifest file without any transformations.
+	UploadAddonAssets bool
```

Comment:
Do we need an additional flag? I thought that `svc package --upload-assets` would basically force the local paths to be uploaded

Reply:
does this help? https://github.com/aws/copilot-cli/pull/3820#discussion_r931466274. i don't think we need any additional flags